lib/tasks/build.js

```diff
@@ -61,5 +62,9 @@ module.exports = Task.extend({
 
         throw err;
       });
+  },
+
+  onInterrupt: function() {
+    return RSVP.Promise.reject(1);
```

Comment:
I'll try `process.exit(code)` approach as you suggested. Seems like I need to better understand how does `capture-exit` actually works